packages/tsconfig/base.json

```diff
@@ -11,7 +11,7 @@
     "noEmit": true,
     "composite": false,
     "target": "ESNext",
-    "module": "Preserve",
+    "module": "ESNext",
     "moduleResolution": "bundler",
```

Comment:
Preserve + Bundler throws an error.

Reply:
This doesn't track. On the contrary, by the TS official docs, if `module` is `preserve`, `moduleResolution` should default to `bundler`.
https://www.typescriptlang.org/tsconfig/#moduleResolution
I'ìll check this for myself later.

Reply:
It's very possible my editor was wrong.

Reply:
yup. just tested this on my end. No errors with "preserve" turned on.

Reply:
Yeah, it's really annoying, sometimes my editor gets confused about tsconfigs. Seems to work fine now.